src/move/econia/sources/incentives.move

```diff
@@ -566,17 +566,17 @@ module econia::incentives {
         IncentiveParameters,
         IntegratorFeeStores
     {
-        // Borrow mutable reference to integrator fee stores map for
-        // given quote coin type.
-        let integrator_fee_stores_map_ref_mut =
-            &mut borrow_global_mut<IntegratorFeeStores<QuoteCoinType>>(
+        // Immutably borrow integrator fee stores map for given quote
+        // coin type.
+        let integrator_fee_stores_map_ref =
```

Comment:
Maybe they should add this as a warning in the move compiler: unused mutable attribute

Reply:
Agreed